Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py

```diff
@@ -350,7 +351,41 @@ def testSpiroAndBridgeheads(self):
     self.assertEqual(len(sa),2)
     self.assertEqual(sorted(sa),[6,9])
     
-
-
+  def testNumRotatableBonds(self):
+    self.assertEquals(-1,
+           int(rdMD.NumRotatableBondsOptions.Default))
+    self.assertEquals(0,
+           int(rdMD.NumRotatableBondsOptions.NonStrict))
+    self.assertEquals(1,
+           int(rdMD.NumRotatableBondsOptions.Strict))
+    self.assertEquals(2,
```

Comment:
They were when I was debugging, as it turns out the order that the variants of CalcNumRotatableBonds is defined in boost is actually important!